data/iam-Rps.json

```
@@ -0,0 +1,70 @@
+{
+  "name": "Rishabh Pratap Singh",
+  "bio": "Open Source Contributor |**DevOps Enthusiast** |",
+  "tags": [
+    "Open Source",
+    "DevOps",
+    "Kubernetes",
+    "Cloud",
+    "Service Mesh",
+    "CI/CD",
+    "Git",
+    "YAML",
+    "Golang",
+    "AWS"
+    "LINUX"
```

Please add semicolons over here
```suggestion
    "AWS",
    "LINUX",
```